### PR TITLE
Prometheus config generation

### PR DESCRIPTION
**Pull request by anti-ckp**
@qu0b  I haven't tested this in our system yet so it's not ready for merge, but could you look at how I've reworked the launcher scripts to see if I didn't accidentally screw up anything deneb related?

I changed the `if IS_DENEB` check to just append to the arguments array rather than making a new one, so that it's easier to see the differences. I also made them alphabetical.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ rebuild-bootstrapper:
 	docker build --no-cache -t ethereum-testnet-bootstrapper -f bootstrapper.Dockerfile .
 
 build-config:
-	docker build --build-arg="CONFIG_PATH=$(config)" -t etb-mainnet-config -f config.Dockerfile .
+	docker build -t etb-mainnet-config -f config.Dockerfile .
```
Comment (anti-stefan) [inline Makefile]: I am using the build-arg for my dencun builds.

File: config.Dockerfile [hunk 2]
```diff
@@ -1,22 +1,8 @@
...
-COPY --from=builder /source/configs /configs 
-COPY --from=builder /source/entrypoint.sh /entrypoint.sh 
-COPY --from=builder /source/docker-compose.yaml /docker-compose.yaml 
-# ADD deps deps
-# ADD src src
-# ADD data data
-# ADD configs configs
-# ADD entrypoint.sh entrypoint.sh
-# ADD docker-compose.yaml docker-compose.yaml
+ADD deps deps
```
Comment (anti-stefan) [inline config.Dockerfile]: This is bad practice as it requires you to build the docker-compose on your local machine in a different environment.

**Review by anti-stefan — CHANGES_REQUESTED** [review]
I've left two comments for you otherwise it looks good from what I can see.

File: Makefile (hunk 1 above)
Comment (anti-ckp) [reply Makefile]: okay I've restored it

File: config.Dockerfile (hunk 2 above)
Comment (anti-ckp) [reply config.Dockerfile]: reverted

**Comment by anti-ckp** [comment]
@anti-stefan This is provisionally ready for merge into deneb, could you give it a look and tell me if there's anything I've overlooked, particularly in the launchers.

**Comment by anti-ckp** [comment]
let me know if anything is confusing, I know it's a big set of changes

**Pull request merged**